### PR TITLE
video_fixed

### PR DESCRIPTION
**Pull request by serglaska**

### CODE REVIEW

File: src/components/PageLayout.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import React, { Fragment } from 'react';
 // components
 import Footer from './Footer';
 import Header from './Header';
+import VideoSection from './VideoDemo';
```
Comment (DmitriyBodlev) [inline src/components/PageLayout.js]: rename file to VideoSection

File: src/components/VideoDemo.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// /////////////////////////////////////////////////////////////////
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: // components

File: src/components/VideoDemo.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { OurDemo, UnderLine, OnlyVideoStyle } from './ui';
+// /////////////////////////////////////////////////////////////////
+export const VideoSection = () => (
+  <OurDemo>
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: OurDemo rename to VideoWrapper

File: src/components/VideoDemo.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <p> Checkout Our Demo </p>
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: Add Comment `// TODO: use Title component when it will be ready`

File: src/components/VideoDemo.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    <UnderLine />
+    <OnlyVideo />
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: remove this and move content from this component here

File: src/components/VideoDemo.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  </OurDemo>
+);
+// /////////////////////////////////////////////////////////////////
+export const OnlyVideo = () => (
+  <fragment>
+    <OnlyVideoStyle>
+      <video
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: Make Gloabal Video component

File: src/components/VideoDemo.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  <OurDemo>
+    <p> Checkout Our Demo </p>
+    <UnderLine />
+    <OnlyVideo />
+  </OurDemo>
+);
+// /////////////////////////////////////////////////////////////////
+export const OnlyVideo = () => (
+  <fragment>
+    <OnlyVideoStyle>
```
Comment (DmitriyBodlev) [inline src/components/VideoDemo.js]: use Global Box component

File: src/components/ui.js [hunk 8]
```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
   }
 `;
 
+// ///////////////////////////////////////////////////////////
+// Vidoe section
+export const OurDemo = styled(Flex)`
+ display: flex;
```
Comment (DmitriyBodlev) [inline src/components/ui.js]: Move all styles in props

File: src/components/ui.js [hunk 9]
```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
...
+ color: #FBFBFB;
+ font-size: 48px;
+ padding-top: 37px;
+ text-align: center;
+ align-items: center;
+ flex-direction: column;
+ background-color: #363135;
+
+ ${createMaxWithMediaQuery(GC.FIRST_UI_BREAKPOINT)} {
```
Comment (DmitriyBodlev) [inline src/components/ui.js]: right tabulation(one tabulation == two spaces)

File: src/components/ui.js [hunk 10]
```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
...
+    font-size: 28px;
+  }
+
+ ${createMaxWithMediaQuery(GC.SECOND_UI_BREAKPOINT)} {
+    font-size: 18px;
+  }
+
+`;
+
+export const UnderLine = styled.div`
```
Comment (DmitriyBodlev) [inline src/components/ui.js:53]: add comment `//TODO: Delete when Title component will  be ready`

File: src/components/ui.js [hunk 11]
```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
...
+ width: 95px;
+ height: 9px;
+ margin: 10px 0;
+ border-radius: 20px;
+ background-color: #9D8C70;
+`;
+
+export const OnlyVideoStyle = styled.div`
```
Comment (DmitriyBodlev) [inline src/components/ui.js]: delete it and use there Flex component with prop `p='30px 0'`

File: src/components/ui.js (hunk 9 above)
Comment (DmitriyBodlev) [reply src/components/ui.js]: ${createMinWithMediaQuery(GC.FIRST_UI_BREAKPOINT)} {
    font-size: 36px;
  }

File: src/components/ui.js [hunk 12]
```diff
@@ -42,6 +42,41 @@ export const HeaderChangeLocaleWrapper = styled(Box)`
...
+ text-align: center;
+ align-items: center;
+ flex-direction: column;
+ background-color: #363135;
+
+ ${createMaxWithMediaQuery(GC.FIRST_UI_BREAKPOINT)} {
+    font-size: 28px;
+  }
+
+ ${createMaxWithMediaQuery(GC.SECOND_UI_BREAKPOINT)} {
```
Comment (DmitriyBodlev) [inline src/components/ui.js]: delete

File: src/constants/index.js [hunk 13]
```diff
@@ -10,3 +10,6 @@ export const ROUTE_TERMS = '/terms';
 export const ROUTE_PRIVACY = '/privacy';
 
 export const FIRST_UI_BREAKPOINT = 640;
+// ////////////////////////////////////
+// by sergii second_ui_breakpoint
+export const SECOND_UI_BREAKPOINT = 320;
```
Comment (DmitriyBodlev) [inline src/constants/index.js]: delete this 3 lines

File: src/components/PageLayout.js (hunk 1 above)
Comment (serglaska) [reply src/components/PageLayout.js]: +

File: src/components/VideoDemo.js (hunk 2 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/VideoDemo.js (hunk 3 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/VideoDemo.js (hunk 4 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/VideoDemo.js (hunk 5 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/VideoDemo.js (hunk 6 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/VideoDemo.js (hunk 7 above)
Comment (serglaska) [reply src/components/VideoDemo.js]: +

File: src/components/ui.js (hunk 9 above)
Comment (serglaska) [reply src/components/ui.js]: +

File: src/components/ui.js (hunk 8 above)
Comment (serglaska) [reply src/components/ui.js]: +

File: src/components/ui.js (hunk 10 above)
Comment (serglaska) [reply src/components/ui.js:53]: +

File: src/components/ui.js (hunk 11 above)
Comment (serglaska) [reply src/components/ui.js]: +

File: src/components/ui.js (hunk 12 above)
Comment (serglaska) [reply src/components/ui.js]: +

File: src/constants/index.js (hunk 13 above)
Comment (serglaska) [reply src/constants/index.js]: +

File: src/components/VideoSection.js [hunk 14]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+// components
+import { VideoWrapper, UnderLine } from './ui';
+// /////////////////////////////////////////////////////////////////
+import { Video } from '../ui/index';
+// TODO: use Title component when it will be ready
```
Comment (VitaliyAndrushko) [inline src/components/VideoSection.js:6]: Add comment -> TODO: with theme and locale

File: src/components/PageLayout.js [hunk 15]
```diff
@@ -2,12 +2,14 @@ import React, { Fragment } from 'react';
...
 import Footer from './Footer';
 import Header from './Header';
+import VideoSection from './VideoSection';
 //  /////////////////////////////////////////////////////////////////////////////////////////////////
 
 export const PageLayout = props => (
   <Fragment>
     <Header {...props} />
     <div>{props.children}</div>
+    <VideoSection {...props} />
```
Comment (VitaliyAndrushko) [inline src/components/PageLayout.js:12]: it should be inside MainPage, not here

**Pull request closed without merging**